### PR TITLE
11612 ir release note

### PR DESCRIPTION
**Pull request by stevsmit**
Version(s):
4.17+

Issue:
https://issues.redhat.com/browse/OSDOCS-11612

Link to docs preview:
https://80022--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html#ocp-4-17-registry

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-17-release-notes.adoc [hunk 1]
```diff
@@ -441,6 +441,13 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-4-17-registry"]
 === Registry
 
+[id="ocp-4-17-chunksizemib-configuration-parameter"]
+==== New chunkSizeMiB configuration parameter for AWS S3 registry storage
```
Comment (xiuwang) [inline release_notes/ocp-4-17-release-notes.adoc:445]: chunkSizeMiB is not only for aws s3, it could be used for the s3 api compatible storage

**Comment by xiuwang** [comment]
LGTM

File: release_notes/ocp-4-17-release-notes.adoc [hunk 2]
```diff
@@ -441,6 +441,13 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-4-17-registry"]
 === Registry
 
+[id="ocp-4-17-chunksizemib-configuration-parameter"]
+==== New chunkSizeMiB configuration parameter for S3 registry storage
+
+A new configuration parameter, `chunkSizeMiB`, is now available for deployments utilizing S3 API-compatible backend storage. When configured, it determines the size of the multipart upload chunks for the S3 API. This parameter is optional. The default value is `10` MiB, with a minimum of `5` MiB.
```
Comment (tmalove) [inline release_notes/ocp-4-17-release-notes.adoc:447]:
```suggestion
A new configuration parameter, `chunkSizeMiB`, is now available for deployments using S3 API-compatible backend storage. When configured, it determines the size of the multipart upload chunks for the S3 API. This parameter is optional. The default value is `10` MiB, with a minimum of `5` MiB.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#u
Also, consider: "A new, optional parameter..."

**Review by tmalove — APPROVED** [review]

**Pull request merged**